config/upstart.example/tfb.conf

```diff
@@ -0,0 +1,18 @@
+# /etc/init/tfb.conf
+env TFB_REPOPARENT="/private"
+env TFB_REPONAME="FrameworkBenchmarks"
+env TFB_REPOURI="git@github.com:ashawnbandy-te-tfb/FrameworkBenchmarks.git"
```

Comment:
I guess you can ignore this comment. After actually setting up continuous benchmarking on Server Central I realized that we need to be using your repo until this is merged into master on the base repo.